### PR TITLE
Shynet doc

### PR DESCRIPTION
**Pull request by ishachopra0401**
## What do these changes do?

Added Shynet doc

### CODE REVIEW

File: docs/destinations/streaming-destinations/shynet.mdx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+## Page 
+
+The <Link to="/event-spec/standard-events/page">`page`</Link> call lets you record your website's page views with any additional relevant information about the viewed page.
+
+### Cloud mode
+
+A sample `page` call for <Link to="/destinations/rudderstack-connection-modes/#cloud-mode">cloud mode</Link> is shown below:
+
+```javascript
```
Comment (ishachopra0401) [inline docs/destinations/streaming-destinations/shynet.mdx:72]: @aashishmalik sample code required here

File: docs/destinations/streaming-destinations/shynet.mdx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+1. From your [RudderStack dashboard](https://app.rudderstack.com/), add a source. Then, from the list of destinations, select **Shynet**.
+2. Assign a name to the destination and click on **Continue**.
+
+## Connection settings
+
+To successfully configure Shynet as a destination, you will need to configure the following settings:
+
+<img src="../../assets/event-stream-destinations/shynet-connection-settings.png" alt="Shynet connection settings"/>
+
+- **Heart Beat**: Enter the time (in ms) to send a ping request. The default value is 5000 ms.
```
Comment (ItsSudip) [inline docs/destinations/streaming-destinations/shynet.mdx:37]: This is a device mode setting only. Can we mention it under the web SDK setting?

File: docs/destinations/streaming-destinations/shynet.mdx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+```javascript
+
+```
+
+The following table lists the mapping for the **required** property between RudderStack and Shynet:
+
+| RudderStack property | Shynet property | Data type | 
+| :----------------- | :--------------- |:--------------- |
+| `context.page.loadTime`        | `loadTime`       | Integer |
```
Comment (ItsSudip) [inline docs/destinations/streaming-destinations/shynet.mdx:80]:
```suggestion
| `properties.loadTime`        | `loadTime`       | Integer |
```

File: docs/destinations/streaming-destinations/shynet.mdx [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+### Cloud mode
+
+A sample `page` call for <Link to="/destinations/rudderstack-connection-modes/#cloud-mode">cloud mode</Link> is shown below:
+
+```javascript
+
+```
+
+The following table lists the mapping for the **required** property between RudderStack and Shynet:
```
Comment (ItsSudip) [inline docs/destinations/streaming-destinations/shynet.mdx:76]: I don't think there are any required fields @aashishmalik can you please confirm?

File: docs/destinations/streaming-destinations/shynet.mdx [hunk 5]
```diff
@@ -0,0 +1,100 @@
+---
+title: "Shynet"
+description: Step-by-step guide on sending your event data from RudderStack to Shynet.
+---
+
+[Shynet](https://github.com/milesmcc/shynet) is an open source web analytics tool which works without cookies and is privacy-friendly.
```
Comment (ItsSudip) [inline docs/destinations/streaming-destinations/shynet.mdx:6]: Shynet is a self-hosted platform, there is not any website as well. We should mention this information somewhere.

**Review by ItsSudip — CHANGES_REQUESTED** [review]

**Comment by ishachopra0401** [comment]
@aashishmalik Can you please respond to the tagged comments? As this is already released in prod, we are looking to close this asap. cc @ItsSudip

File: docs/destinations/streaming-destinations/shynet.mdx (hunk 4 above)
Comment (aashishmalik) [reply docs/destinations/streaming-destinations/shynet.mdx:77]: updated doc

**Review by ItsSudip — APPROVED** [review]

**Review by aashishmalik — APPROVED** [review]

File: docs/destinations/streaming-destinations/shynet.mdx [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+1. From your [RudderStack dashboard](https://app.rudderstack.com/), add a source. Then, from the list of destinations, select **Shynet**.
+2. Assign a name to the destination and click on **Continue**.
+
+## Connection settings
+
+To successfully configure Shynet as a destination, you will need to configure the following settings:
+
+<img src="../../assets/event-stream-destinations/shynet-connection-settings.png" alt="Shynet connection settings"/>
+
+- **Heart Beat**: This is a **device mode** only setting where you need to enter the time (in ms) to send a ping request. The default value is 5000 ms.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/shynet.mdx:38]:
```suggestion
- **Heart Beat**: This is a **device mode** only setting. Enter the time (in ms) to send a ping request. The default value is 5000 ms.
```

File: docs/destinations/streaming-destinations/shynet.mdx [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+rudderanalytics.page("HomePage", {
+  loadTime: 55,
+});
+```
+
+The following table lists the mappings between RudderStack and Shynet properties:
+
+| RudderStack property | Shynet property | Presence | Data type | 
+| :----------------- | :--------------- |:--------------- |:--------------- |
+| `context.page.url`        | `location`      | Required | String |
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/shynet.mdx:83]: @aashishmalik @ItsSudip in the sample snippet above, we have `loadTime` which is an optional property but not `url` which is required. Please confirm if it should be added. cc @ishachopra0401

Comment (ItsSudip) [reply docs/destinations/streaming-destinations/shynet.mdx:83]: Our javascript SDK automatically tracks the URL. But we should update the sample snippet with the required property.

Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/shynet.mdx:89]: cc @ishachopra0401

**Review by ameypv-rudder — APPROVED** [review]

**Pull request merged**